core/templates/domain/exploration/ExplorationMetadataObjectFactorySpec.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for ExplorationMetadataObjectFactory.
+ */
+
+import { TestBed } from '@angular/core/testing';
+
+import { ExplorationMetadataObjectFactory } from
+  'domain/exploration/ExplorationMetadataObjectFactory';
+
+
+describe('Exploration Metadata object factory', () => {
+  var _sampleExplorationMetadata = null;
```

Comment:
Use `const` or `let` when testing Angular 8 files. Also I think it's ok to not using the underscore on the name.

Reply:
Removed the underscore, and changed var to let.